LOG.tbasicu

```diff
@@ -150,33 +155,32 @@ Type tLogThis
     Local sTagF as String
     Local sDT as String
     Local sDTF as String
-    Local successFile as Number
-    Local logWhatTitleFile = logWhatTitle
+    Local logWhatTitleFile as String = logWhatTitle
     
-    '[t] Workaround for Switch$ error ( see https://www.thinbasic.com/community/project.php?issueid=616 ) ...
-    local TagDebug = me.TagDebug 
-    local TagInfo = me.TagInfo
-    local TagNotice = me.TagNotice
-    local TagWarn = me.TagWarn
-    local TagErr = me.TagErr
-    local TagCrit = me.TagCrit
-    local TagAlert = me.TagAlert
-    local TagPanic = me.TagPanic
+    #region "Workaround for Switch$ error" '[t] Workaround for Switch$ error ( see https://www.thinbasic.com/community/project.php?issueid=616 )
```

Comment:
```suggestion
    #region "Workaround for Switch$ error"
'[t] Workaround for Switch$ error ( see https://www.thinbasic.com/community/project.php?issueid=616 ) 
```

Reply:
Updated on all relevant places, thank you!